example/src/Client/utils/AccountsManager.ts

```diff
@@ -0,0 +1,135 @@
+import { generateKeyFromPrivateKey, Key } from "@ironfish/rust-nodejs";
+import { NoteEncrypted } from "@ironfish/sdk/build/src/primitives/noteEncrypted";
```

Comment:
We should try and remove all references to the sdk in this client as much as possible, since integrators working here will not have access to SDK functions (unless they are in typescript, which is unlikely)